### PR TITLE
DOC: Updates for Windows

### PR DESCRIPTION
**Pull request by cgohlke**

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -47,40 +45,33 @@ If you are on Mac OSX you can probably install matplotlib binaries using the
...
-<http://www.microsoft.com/en-us/download/details.aspx?id=48145>`_
-needs to be installed.
-In case Python 2.7 to 3.4 are not installed for all users (not the default),
+Enthought Canopy, or Continuum Anaconda, which have matplotlib and many
+of its dependencies, plus other useful packages, preinstalled.
+
+For `standard Python <http://www.python.org/download/>`_ installations,
+install matplotlib using `pip <https://pypi.python.org/pypi/pip/>`_::
+
+    python -m pip install -U pip setuptools
```
Comment (story645) [inline INSTALL]: do you still need to update setuptools for a straight matplotlib install?

File: INSTALL [hunk 2]
```diff
@@ -47,40 +45,33 @@ If you are on Mac OSX you can probably install matplotlib binaries using the
...
+    python -m pip install matplotlib
+
+In case Python 2.7 or 3.4 are not installed for all users (not the default),
```
Comment (story645) [inline INSTALL]: double negative makes it unclear as whether the default behavior is that they're install for users or not. Rephrase to _default is installed for all users_

**Comment by story645** [comment]
Since you're working on the front page, how do you feel about sorting out issue #1069 ?

File: INSTALL [hunk 3]
```diff
@@ -47,40 +45,33 @@ If you are on Mac OSX you can probably install matplotlib binaries using the
...
-`matplotlib <http://pypi.python.org/pypi/matplotlib/>`_.
-
-For Python 3.5 the `Visual C++ Redistributable for Visual Studio 2015
-<http://www.microsoft.com/en-us/download/details.aspx?id=48145>`_
-needs to be installed.
-In case Python 2.7 to 3.4 are not installed for all users (not the default),
+Enthought Canopy, or Continuum Anaconda, which have matplotlib and many
+of its dependencies, plus other useful packages, preinstalled.
+
+For `standard Python <http://www.python.org/download/>`_ installations,
```
Comment (QuLogic) [inline INSTALL]: Should change to https.

File: INSTALL [hunk 4]
```diff
@@ -105,18 +96,19 @@ For other backends you may need to install
...
-the `PyPI download page <http://pypi.python.org/pypi/matplotlib/>`_ do
-not contain test data or example code.  If you want to try the many
-demos that come in the matplotlib source distribution, download the
-:file:`*.tar.gz` file and look in the :file:`examples` subdirectory.
-To run the test suite, copy the :file:`lib\matplotlib\tests` and
-:file:`lib\mpl_toolkits\tests` directories from the source
-distribution to :file:`sys.prefix\Lib\site-packages\matplotlib` and
-:file:`sys.prefix\Lib\site-packages\mpl_toolkits` respectively, and
+The Windows wheels (:file:`*.whl`) on the `PyPI download page
+<http://pypi.python.org/pypi/matplotlib/>`_ do not contain test data
```
Comment (QuLogic) [inline INSTALL]: Also should be changed to https.

File: INSTALL (hunk 1 above)
Comment (cgohlke) [inline INSTALL]: Older Python 2.7 versions may not have setuptools installed.

**Pull request merged**